provider/aws/aws_test.go

```diff
@@ -353,6 +353,12 @@ func TestAWSRecords(t *testing.T) {
 			TTL:             aws.Int64(recordTTL),
 			ResourceRecords: []*route53.ResourceRecord{{Value: aws.String("8.8.8.8")}},
 		},
+		{
+			Name:            aws.String("AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA.ext-dns-test-2.teapot.zalan.do."),
```

Comment:
Should we add a test for a label >63 chars? I've been using `this-is-an-exceedingly-long-label-that-external-dns-should-reject` locally which is 65.